packages/@tinacms/cli/src/cmds/init/index.ts

```diff
@@ -95,13 +95,13 @@ export async function tinaSetup(ctx: any, next: () => void, options) {
 
   // 2. Create a Tina Provider
   if (
-    !fs.pathExistsSync(TinaProviderFolder) &&
+    !fs.pathExistsSync(TinaFolder) &&
     !fs.existsSync(TinaProviderPath) &&
-    !fs.existsSync(TinaProviderPathIndex)
+    !fs.existsSync(TinaDynamicProvider)
   ) {
-    fs.mkdirpSync(TinaProviderFolder)
+    fs.mkdirpSync(TinaFolder)
     fs.writeFileSync(TinaProviderPath, TinaProvider)
-    fs.writeFileSync(TinaProviderPathIndex, TinaProviderIndex)
+    fs.writeFileSync(TinaDynamicProvider, TinaProviderDynamic)
```

Comment:
@logan-anderson You might still be working on this, but I see we sometimes call it `TinaProviderDynamic` & sometimes `TinaDynamicProvider`